projects/components/src/navigation/navigation-list.component.scss

```diff
@@ -8,24 +8,23 @@
 
   height: 100%;
   width: 100%;
-  padding: 24px 5px 5px 8px;
```

Comment:
Padding will be provided in parent component. Right padding is required for resize icon